### PR TITLE
Firefox 51 beta 3 testday invitation

### PR DESCRIPTION
**Pull request by prasanthp96**

### CODE REVIEW

File: blog/_posts/2016-11-19-call-for-Testing-day-firefox 51b3.markdown [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title:  "Firefox 51.0b3 Beta Test day invitation"
```
Comment (iamVP7) [inline blog/_posts/2016-11-19-call-for-Testing-day-firefox 51b3.markdown]: Check file Name **firefox 51b3.markdown**  there is no -

File: blog/_posts/2016-11-19-call-for-Testing-day-firefox 51b3.markdown [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+<img height="20%" width="100%" src="https://pbs.twimg.com/media/ChI8q4UU8AAcw9t.jpg">
+
+**Hello Everyone!**
+
+ Monthly twice, the <a href="https://quality.mozilla.org/">Quality Assurance</a> team of Mozilla conducts "test day" in which people are invited to download and test the unstable Aurora and Nightly versions of Firefox Browser for Desktop for bugs. The test day involves executing the given testcases and recording the output. 
+
+For the past 4 months many contributors from India actively participated in the testdays, and here is the list of all the testday results.
+
+	<a href="http://foxprasanth.github.io/events/2016/07/10/MozTN-in-testingdays/">Firefox 48 beta 3</a>
```
Comment (iamVP7) [inline blog/_posts/2016-11-19-call-for-Testing-day-firefox 51b3.markdown]: Can we have different way of adding hyper links

File: blog/_posts/2016-11-19-call-for-Testing-day-firefox 51b3.markdown [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+<h5>FAQ</h5>
+
+**Any previous experience is required ?**
+
+No previous experience is required for Testing.
+
+
+**Prerequisite?**
+
+Bugzilla Account- to access bugs.
```
Comment (iamVP7) [inline blog/_posts/2016-11-19-call-for-Testing-day-firefox 51b3.markdown]: Can we ask people to do RSVP their twitter handle so they can share their interest

File: blog/_posts/2016-11-19-call-for-Testing-day-firefox 51b3.markdown (hunk 1 above)
Comment (prasanthp96) [reply blog/_posts/2016-11-19-call-for-Testing-day-firefox 51b3.markdown]: got it.

File: blog/_posts/2016-11-19-call-for-Testing-day-firefox 51b3.markdown (hunk 3 above)
Comment (prasanthp96) [reply blog/_posts/2016-11-19-call-for-Testing-day-firefox 51b3.markdown]: sure

File: blog/_posts/2016-11-19-call-for-Testing-day-firefox 51b3.markdown (hunk 2 above)
Comment (prasanthp96) [reply blog/_posts/2016-11-19-call-for-Testing-day-firefox 51b3.markdown]: different way means?

**Comment by prasanthp96** [comment]
Added RSVP notification and the renamed the file @iamVP7

File: blog/_posts/2016-11-19-call-for-Testing-day-firefox-51b3.markdown [hunk 4]
```diff
@@ -0,0 +1,43 @@
+---
+title:  "Firefox 51.0b3 Beta Test day invitation"
+date:   2016-11-19 21:50:00 +0530
+description: Mozilla Tamilnadu Community Testing day invitation
+categories: blog
+tags: [firefoxqa]
+author: prasanthp96
+---
+
+<img height="20%" width="100%" src="https://pbs.twimg.com/media/ChI8q4UU8AAcw9t.jpg">
```
Comment (iamVP7) [inline blog/_posts/2016-11-19-call-for-Testing-day-firefox-51b3.markdown]: Dont add image from outside. Check previous post and see there is image source.. and see the way image added

Comment (prasanthp96) [reply blog/_posts/2016-11-19-call-for-Testing-day-firefox-51b3.markdown]: okay bro :+1:

**Pull request merged**